maintenance/Caddyfile

```diff
@@ -1,16 +1,19 @@
-0.0.0.0:2015
-
-root /var/www/html
-
-log stdout
-
-errors stdout
+{
+    storage file_system {
+        root /var/www/html
+        http_port 2016
+        https_port 2015
+    }
+}
 
-rewrite / {
-  regexp .*
-  to /maintenance.html
+:2015
+root * /var/www/html
+file_server {
+  index maintenance.html
 }
 
+log
+
```

Comment:
what is a Caddy/Caddyfile?

Reply:
Caddy is another web server stack. You've heard of Apache probably? Caddy is like that. Caddy is nice because it does https automatically for you. Not sure if you've ever had to set up https manually, but its a pain.

https://caddyserver.com/docs/

Reply:
Ah okay thank you!